### PR TITLE
updating rtl and ltr as constant

### LINKED ISSUE #3362: i18n `getDirection()` helper should rely on constants
In `core/i18n/utils`, the `getDirection()` function should not use hardcoded values for `ltr` and `rtl` because then we need to test with the hardcoded values when using this helper:

https://github.com/mozilla/addons-frontend/blob/92b83273ba0c86ab750e92faf1c8e5a50c033aa8/src/core/i18n/utils.js#L104-L106

We should rather have `LTR` and `RTL` as constants, and use them. When working on this issue, please "grep" for `'rtl'`, `'ltr'` and `getDirection()` and update the code accordingly.

### PR DESCRIPTION
**Pull request by championshuttler**
[WIP]
Fixes #3362

how to approach with scss files, should I import constants from `core/constants` or any better approach

### CODE REVIEW

**Comment by willdurand** [comment]
> how to approach with scss files, should I import constants from core/constants or any better approach

don't touch the SCSS code I'd say.

**Comment by championshuttler** [comment]
@willdurand haha ok 😜

**Comment by championshuttler** [comment]
Then I think it's ready for first review 😉

File: src/core/constants.js [hunk 1]
```diff
@@ -299,3 +299,6 @@ export const RATINGS_MODERATE = 'Ratings:Moderate';
 export const USERS_EDIT = 'Users:Edit';
 // Super powers. It means absolutely all permissions.
 export const ALL_SUPER_POWERS = '*:*';
+
```
Comment (championshuttler) [inline src/core/constants.js]: Should I add a comment here like `Left-to-right and right-to-left variables` .

Comment (willdurand) [reply src/core/constants.js]: nope

**Comment by willdurand** [comment]
There is one occurrence that needs to be changed in `src/core/components/ServerHtml/index.js`

**Comment by willdurand** [comment]
@championshuttler There is one occurrence that needs to be changed in `src/core/components/ServerHtml/index.js`

**Comment by championshuttler** [comment]
Done , next level stupid i am :man_facepalming:

**Pull request merged**